src/scales/scale.ts

```diff
@@ -32,6 +32,10 @@ module Plottable {
       this._extentsProviders = new Utils.Set<Scales.ExtentsProvider<D>>();
     }
 
+    public getExtentFromDomain(data: D[]): D[] {
```

Comment:
We should probably:
* use a name without "get" in it
* Use "domain" in a way that's consistent with its use on `Scale`

In particular, since the argument is named `data` I would expect the method to be named something like `extentFromData()`.